core/test/financial-templates/PricelessPositionManager.js

```diff
@@ -940,23 +964,37 @@ contract("PricelessPositionManager", function(accounts) {
     const sponsorFinalSynthetic = await tokenCurrency.balanceOf(sponsor);
 
     // The token Sponsor should gain the value of their synthetics in underlying
-    // + their excess collateral from the over collateralization in their position
-    // Excess collateral = 30 - 20 * 1.2 - 1 - roundingErrors <= 5
+    // + their excess collateral from the over collateralization in their position.
+    // Excess collateral should be = rawCollateral - fees - tokensOutstanding * price = 30 - 1 - (20 * 1.2) = 5
+    // However, recall that `totalPositionCollateral = (30 * 0.966666666666666666 = 28.999...98)` which gets truncated to 28.
+    // So, the excess collateral becomes 28 - (20 * 1.2) = 4
+    // The value of the remaining synthetic tokens = 10 * 1.2 = 12.
+    // So, we will attempt to withdraw (12 + 4) tokens from the contract.
+    // We need to decrease `rawCollateral` by `16 / 0.966666666666666666 ~= 16.5`
+    // which gets truncated to 16.
+    // Recall previously that rawCollateral was last set to 18, so `totalPositionCollateral = (18-16) * 0.966666666666666666 ~= 1.97`
+    // which gets truncated to 1.
+    // The previous totalPositionCollateral was 17, so we will withdraw (17-1) = 16 tokens instead of the 17 as the user expected.
+    assert.equal((await pricelessPositionManager.totalPositionCollateral()).toString(), "1");
+    assert.equal((await pricelessPositionManager.rawTotalPositionCollateral()).toString(), "2");
+    const expectedSponsorCollateralSynthetic = toBN("11");
```

Comment:
This is excellent and your comments perfectly show how this PR's change to under credit in favour of rounding results in this single wei less in the transfer.